db/migrate/20230325001538_add_descriptive_title_to_questions.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptiveTitleToQuestions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :questions, :descriptiveTitle, :string
```

Comment:
this needs to be all lowercase.

Reply:
Should it be descriptive_title? or descriptivetitle? I'm not sure if the first is allowed